extension/rikaichan.ts

```diff
@@ -263,7 +263,10 @@ class RcxMain {
 
     const m = this.showMode;
     let e: DictEntryData | null = null;
-
+    if (text === '') {
```

Comment:
Add a test for this branch (as you probably saw from the coverage report. :) )

Reply:
Yeah, Let me do some digging and thinking for your above question and also handle the tests for this if this is the approach we end up going with 👍

Reply:
@melink14 , I moved the quick catch to the `background.ts`. I also added tests to cover 99% of the file. 

Had trouble covering this line. [Here](https://github.com/melink14/rikaikun/blob/d2a52a41a8619c9282e9a995b25cb5f1a2e52ec9/extension/background.ts#L38)

I tried a few different approaches and even tried using a 3rd party package `chai-as-promised` to assist with async testing to no avail. 

I'll leave it to you or circle back later. 👍

Reply:
That sounds good. You should revert the changes to thsi file then I guess!

Also, the additional tests for expanding coverage are of course good but I wouldn't include it in the same PR as one fixing a specific bug. It makes it hard for a reviewer and future reader to know which tests were directly related to the fix and which were just updating code quality in general.

Is it possible to do a quick PR split and keep things very focused?

Reply:
You got it.